### PR TITLE
feat(Composer): Add a composer.json file

### PR DESCRIPTION
**Pull request by e0ipso**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "contentacms/snail",
```
Comment (dawehner) [inline composer.json:2]: I'm wondering whether this should be drupal/snail ...

**Comment by boztek** [comment]
Maybe I don't know enough about how Drupal.org plays with composer but wouldn't drupal/snail correspond to code in https://www.drupal.org/project/snail ? If so there is no code there as far as I can tell so contentacms/snail should be right. Or am I unaware of some github to Drupal.org workflow magic?

**Comment by e0ipso** [comment]
@boztek you are right. We should talk to @justafish to see if she can push the code here into that repository.

**Comment by dawehner** [comment]
@boztek Good point. Let's push it to drupal.org at let people install it from there.

**Comment by dawehner** [comment]
Alright, we no longer need that. Strike!

**Pull request closed without merging**